chat-server/server.js

```diff
@@ -25,6 +26,31 @@ app.get("/", (request, response) => {
   response.sendFile(__dirname + "/index.html");
 });
 
+app.post("/messages", (req, res) => {
+  const newMessage = req.body;
+  messages.push(newMessage);
```

Comment:
Good grasp of getting Create (POST) data! However, does the POST data contain all properties required in the [Data Model](https://github.com/CodeYourFuture/Module-Servers/tree/main/chat-server#data-model) section?